### PR TITLE
Rebuild libhdfs3 2.3.0 with libxml2 2.10

### PR DESCRIPTION
**Pull request by skupr-anaconda**

### CODE REVIEW

**Comment by katietz** [comment]
hmm, for what we need 2.10? we pin in cbc to 2.9

**Comment by katietz** [comment]
àh, I see. it is a rebuild ...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ requirements:
     - libgsasl
     - libntlm
     - libuuid
-    - libxml2
+    - libxml2  {{ libxml2 }}
```
Comment (katietz) [inline recipe/meta.yaml]: please replace libxml2 by 2.10 here. no need to add it to a private cbc

File: recipe/meta.yaml [hunk 2]
```diff
@@ -39,7 +39,7 @@ requirements:
     - libgsasl
```
Comment (katietz) [inline recipe/meta.yaml:37]: in build section the windows variant is missing for patch ... please add.

Comment (katietz) [reply recipe/meta.yaml:37]: please remove the ignore_run_exports for libntlm ... add an abs.yaml file disabling overdepending checks instead

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Done

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:37]: We skip building on `win` that's why there is only unix `patch`

Comment (skupr-anaconda) [reply recipe/meta.yaml:37]: I've added abs.ayml to ignore overdepending warnings

**Review by akabanovs — COMMENTED** [review]
libprotobuf and it's version is defined in aggregate cbc. Should not we add 

={libprotobuf}

in the host section so that future packagers are aware?

Same for libgsasl. It is defined in aggregate cbc and hence here not the latest version is used (1.10.0), but actually 1.8.0. Should not we add to the host section {libgsasl} to reflect this?

**Pull request merged**